src/service_client.cpp

```diff
@@ -40,7 +40,7 @@ void ServiceClient::onRos2Initialized()
   try {
     rclcpp::Node &node = *Ros2Qml::getInstance().node();
     client_ =
-        babel_fish_.create_service_client( node, name_.toStdString(), service_type_.toStdString() );
+        babel_fish_.create_service_client( node, name_.toStdString(), service_type_.toStdString(), qos_.getQoS().get_rmw_qos_profile() );
```

Comment:
I'm not entirely sure why `create_service_client(..)` needs a `rmw_qos_profile_t`, while `create_subscription(..)` et al. accept `rclcpp::QoS`?

Reply:
Is this something that should change in `ros2_babel_fish`? Or is it as-intended?

Reply:
Mainly because the ROS2 service client required the C struct in its constructor but I guess it makes sense to change that in `ros2_babel_fish` to make it more consistent.
I will change it but keep the old function for a while with a deprecation message.